lib/services/ios-project-service.ts

```diff
@@ -704,7 +704,7 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 		return Promise.resolve();
 	}
 
-	private async mergeInfoPlists(projectData: IProjectData): Promise<void> {
+	private async mergeInfoPlists(release: boolean, projectData: IProjectData): Promise<void> {
```

Comment:
According to our coding guidelines, we prefer passing objects with the required boolean properties, which will make the calling of the function more readable. In this case I suggest you to use the `IRelease` interface, for example:
```TypeScript
private async mergeInfoPlists(buildOption: IRelease, projectData: IProjectData): Promise<void> {
```
In the method body you'll check if `buildOptions.release` is true. The caller will pass `{ release: <value> `